dms/models/dms_security_mixin.py

```diff
@@ -287,8 +306,41 @@ def filter_access_groups(self, operation):
             )
             self.env.cr.execute(sql_query, [self.env.user.id])
             ids_with_access += list(map(lambda val: val[0], self.env.cr.fetchall()))
+            # Add related to record_ref_mail_follower_ids
+            if self._table == "dms_directory" and not self.env.context.get(
```

Comment:
This wasn't wrong, but it'd be the common way to check it:

```suggestion
            if self._name == "dms.directory" and not self.env.context.get(
```